src/main/java/edition/academy/seventh/security/response/JwtResponse.java

```diff
@@ -25,8 +24,8 @@
    * Creates JwtResponse with required properties and toked which will be held locally by the client
    * user. Contains {@link GrantedAuthority authorities} defining user privileges.
    *
-   * @param accessToken Generated JSON web token {@link
-   *     edition.academy.seventh.security.jwt.JwtProvider#generateJwtToken(Authentication)}.
+   * @param accessToken Generated JSON web token {@link // *
```

Comment:
Co to za // ?

Reply:
Rozwiązane na gałęzi 273#scalenie_do_mastera
Poprawione ale usunięty znacznik @link ponieważ nie ma dostępu do tej metody z tej klasy (domyślny modyfikator dostępu).